borgmatic/commands/borgmatic.py

```diff
@@ -61,6 +61,7 @@ def run_configuration(config_filename, config, arguments):
         for section_name in ('location', 'storage', 'retention', 'consistency', 'hooks')
     )
     global_arguments = arguments['global']
+    global_arguments.config_paths = used_config_paths
```

Comment:
Will this cause a problem?

Reply:
Okay, I'll work on this next for this PR.

Also, I was thinking of adding the borgmatic version used to create the archive inside this manifest too, so I use the correct config with the correct version. What do you think, and where can I get the current version from? A quick ctrl+f didn't help.